### ISSUE TITLE
Open webview/browser in location outside of manga view? MangaDex Error

### ISSUE THREAD

**Issue by Thionises** [inorichi/tachiyomi-extensions#1035]
I'm on tachiyomi version 0.8.4, and my mangadex version is 1.2.54. I just recently got a new phone and am having an issue where I can't get into mangadex in browser/webview to prevent the 403 error. Normally I'd go to my library and open one of the manga that I had there in webview and then complete the captcha to prevent the 403 error. However, with this new phone and the downloaded app and extension, no manga are popping up at all for mangadex because of the 403 error. Because of this, I can't open webview to solve this issue since it seems that the webview option is only available through a manga's info page. Is it possible to add an option in the catalogue to open extensions in webview/browser to prevent this sort of error? I also tried to go to a manga from mangarock instead of mangadex, and then swap the browser url to mangadex and sign in. However you cannot swap the browser url unless you open in chrome, and when I opened in chrome it no longer counted towards tachiyomi's error Any insight into solving this issue would be great, but I can just use my old phone until this issue gets solve. 

TL DR: 403 error prevents manga from showing up, and because no manga are showing up I can't get into webview to complete the captcha and prevent the 403.

**Comment by CarlosEsco** [comment]
There is already a feature for this in dev version.
